### ISSUE TITLE
[BUG] [Form recognizer] SDK doesn't parse the currency symbol part of the amounts

### ISSUE THREAD

**Issue by andrea-del-popolo** [Azure/azure-sdk-for-java#31775]
**Describe the bug**

Using the model `prebuilt:invoice` I cannot extract the currency symbol for the invoice amount, or for any other amount on the document. The support for the extraction of currency was added in version [4.0.0-beta.4 (2022-02-10)](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md#400-beta4-2022-02-10) (I trace it back [to this PR](https://github.com/Azure/azure-sdk-for-java/pull/26572) by @samvaity) and it is still supported by the FormRecognizer API version 2022-08-31-General Availability

***Exception or Stack Trace***
N/A

**To Reproduce**
- Find an invoice with a supported currency symbol, such as EUR
- Run the SDK method [beginRecognizeInvoices](https://github.com/Azure/azure-sdk-for-java/blob/644f8356ea30fde64c3ad13a18d242557f7f80da/sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/FormRecognizerClient.java#L1290)
- Parsing the result `RecognizedForm.fields` contains only the float number corresponding to the amount, but there is no way to read the currency part in the models exposed in the SDK.

***Code Snippet***
N/A

**Expected behavior**
In version [4.0.0-beta.4 (2022-02-10)](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md#400-beta4-2022-02-10) it was possible to extract the currency part from the InvoiceTotal fields, it's not anymore possible in the latest version. The expected behaviour is to be able to read the currency part as well.

```json
"InvoiceTotal": {
  "type": "currency",
  "valueCurrency": {
	  "currencySymbol": "€",
	  "amount": 110
  },
```

**Screenshots**
Using [the Form Recognizer Studio](https://formrecognizer.appliedai.azure.com/studio/prebuilt?formType=invoice)
From the fields view:
<img width="1432" alt="image" src="https://user-images.githubusercontent.com/48206879/198279358-db864e63-17b7-4a43-8254-992d42f05651.png">
And in the Result view:
<img width="1432" alt="image" src="https://user-images.githubusercontent.com/48206879/198279599-10da9c78-34b4-4c57-93b1-db8157cdb8a8.png">

**Setup (please complete the following information):**
 - Library/Libraries: `com.azure:azure-ai-formrecognizer:4.0.1`

**Additional context**

**Information Checklist**
Kindly make sure that you have added all the following information above and checkoff the required fields otherwise we will treat the issuer as an incomplete report
- [x] Bug Description Added
- [x] Repro Steps Added
- [x] Setup information Added

**Comment by samvaity** [comment]
Hey @andrea-del-popolo, thanks for reporting this.
>Parsing the result RecognizedForm.fields contains only the float number corresponding to the amount, but there is no way to read the currency part in the models exposed in the SDK.

This is expected behavior as [CurrencyValue]() is only supported for FormRecognizer API version 2022-08-31-General Availability which is only supported in the newer clients i.e DocumentAnalysisClient/AsyncClient.
<img width="573" alt="image" src="https://user-images.githubusercontent.com/16845631/198380012-bb2ed402-ea6e-4402-afe5-3d3640aed0b7.png">
The [Migration Guide here](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/formrecognizer/azure-ai-formrecognizer/migration-guide.md) explains this in more detail.

To access the currency field i.e symbol & amount, you can use the `DocumentAnalysisClient` 
`invoicePageFields.get("InvoiceTotal").getValueAsCurrency().getSymbol()`

**Comment by samvaity** [comment]
@@andrea-del-popolo Let me know if this is still unclear or needs more information.

**Comment by andrea-del-popolo** [comment]
> @@andrea-del-popolo Let me know if this is still unclear or needs more information.

@samvaity thank you this is clear now! I couldn't understand at first, but now I realised that between the version beta.5 and 4.0.0 the namespace of `DocumentAnalysisClient` and the other bits moved from `com.azure.ai.formrecognizer.DocumentAnalysisClient` to `com.azure.ai.formrecognizer.documentanalysis.DocumentAnalysisClient`. I ended up importing the older client version in an attempt to find the `DocumentAnalysisClient` 😅 

Thank you for your explanation and prompt answer!
